### PR TITLE
How to send selects

### PR DESCRIPTION
**Pull request by Toricane**
## About this pull request

How to send selects

## Changes

The example of how to create and react to a select did not show how to send a select in the first place. I added an extra line showing how to send a select.

## Checklist

- [ ] I've run the `pre_push.py` script to format and lint code.
- [ ] I've checked this pull request runs on `Python 3.6.X`.
- [ ] This fixes something in [Issues](https://github.com/eunwoo1104/discord-py-slash-command/issues).
    - Issue:
- [ ] This adds something new.
- [ ] There is/are breaking change(s).
- [ ] (If required) Relevant documentation has been updated/added.
- [x] This is not a code change. (README, docs, etc.)

### CODE REVIEW

**Comment by Toricane** [comment]
Made another change to this PR which fixed the example since you can't send a message without content, and explains the limits

File: docs/components.rst [hunk 1]
```diff
@@ -117,11 +117,11 @@ Each button gets a ``custom_id`` (which is always a string), this is a unique id
 What about selects / Dropdowns?
 _______________________________
 
-Yep we support those too. You use them much the same as buttons:
+Yep we support those too. You use them much the same as buttons, with 25 options per select, 1 select in each action row, 5 action rows in a message, totalling 125 options:
```
Comment (LordOfPolls) [inline docs/components.rst]:
```suggestion
Yep we support those too. You use them much the same as buttons. You can only have 1 select per action row, but each select can have up to 25 options in it!
```

I really dont think we should be encouraging pushing selects to their limits with 125 options. That'd be incredibly bad UX

Comment (Toricane) [reply docs/components.rst]: Ok yeah that looks better

**Comment by Toricane** [comment]
Um the check failed:

would reformat discord_slash/utils/manage_components.py
Oh no! 💥 💔 💥
1 file would be reformatted, 14 files would be left unchanged.
Error: Process completed with exit code 1.

There is something wrong with manage_components.py?

**Comment by LordOfPolls** [comment]
Weird it didn't catch that when I merged last. I'll look into it

**Pull request merged**